src/core/mac/mac.cpp

```diff
@@ -340,6 +340,10 @@ void Mac::PerformEnergyScan(void)
     }
     else
     {
+        if(!GetRxOnWhenIdle())
```

Comment:
To resolve pretty check:
```suggestion
        if (!GetRxOnWhenIdle())
```